src/test/regress/sql/upgrade_distributed_table_after.sql

```diff
@@ -1,28 +1,99 @@
-SET search_path TO upgrade_distributed_table_before, public;
+SET search_path TO upgrade_distributed_table_before, public, pg_catalog;
+\d t
+\d tp
+\d t_ab
+\d t2
+\d r
+\d tr
+
+SELECT nextval('pg_dist_shardid_seq') = MAX(shardid)+1 FROM pg_dist_shard;
+SELECT nextval('pg_dist_placement_placementid_seq') = MAX(placementid)+1 FROM pg_dist_placement;
+SELECT nextval('pg_dist_groupid_seq') = MAX(groupid)+1 FROM pg_dist_node;
+SELECT nextval('pg_dist_node_nodeid_seq') = MAX(nodeid)+1 FROM pg_dist_node;
+SELECT nextval('pg_dist_colocationid_seq') = MAX(colocationid)+1 FROM pg_dist_colocation;
+
+-- If this query gives output it means we've added a new sequence that should
+-- possibly be restored after upgrades.
+SELECT sequence_name FROM information_schema.sequences
+  WHERE sequence_name LIKE 'pg_dist_%'
+  AND sequence_name NOT IN (
+    -- these ones are restored above
+    'pg_dist_shardid_seq',
+    'pg_dist_placement_placementid_seq',
+    'pg_dist_groupid_seq',
+    'pg_dist_node_nodeid_seq',
+    'pg_dist_colocationid_seq',
+    -- jobid is not actually stored in tables, so it does not need to be restored
+    'pg_dist_jobid_seq'
```

Comment:
@marcocitus Currently we don't restore jobid after an upgrade. Looking at it's definition that seems reasonable:
https://github.com/citusdata/citus/blob/master/src/backend/distributed/sql/citus--7.0-1.sql#L110-L118

 I wanted to double check that it's indeed fine that we don't restore it.

Reply:
That's fine. We don't actually use that sequence for anything. It should be dropped.

Reply:
I added a commit that drops this sequence.